src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (repeatPassword == null
+                || password == null
+                || password.length() == 0
+                || password.length() < 10
+                || !password.equals(repeatPassword)) {
```

Comment:
Деякі умови перевіряються двічі, краще зробити так:
```suggestion
        if (password == null || password.length() < 10 || !password.equals(repeatPassword)) {
```